HW03/maksim_manzhai/maksim_manzhai_hw_03_t_01.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class Homework3
+  def task1(logs)
+    return '' if logs.empty?
+
+    lines = logs.split(/\n/)
+    lines.find { |line| line.downcase.include?('error') } || ''
```

Comment:
you can do the same in one line
```suggestion
logs.split(/\n/).find { |line| line.downcase.include?('error') } || ''
```